src/version.h

```diff
@@ -4,8 +4,8 @@
  * similar. */
 #define SERVER_NAME "valkey"
 #define SERVER_TITLE "Valkey"
-#define VALKEY_VERSION "255.255.255"
-#define VALKEY_VERSION_NUM 0x00ffffff
+#define VALKEY_VERSION "8.1.0"
+#define VALKEY_VERSION_NUM 0x00080100
```

Comment:
i think we have a different version for RC? something like this:
```
// 8.0 RC1
#define VALKEY_VERSION "7.9.240"
#define VALKEY_VERSION_NUM 0x000709f0
```

Reply:
O.K will try using 8.0.240

Reply:
@enjoy-binbin changing this breaks many tests as the new commands/subcommands (eg IFEQ) are written in the json with 8.1 support...

Reply:
https://github.com/valkey-io/valkey/issues/1650

We discussed this in the meeting last week. I think we were leaning towards using 8.1.0 in release candidates and GA, but maybe we didn't make a decision... Using 8.0.240 was the other alternative. We said that it's unlikely that users depend on this in release candidates though.

@enjoy-binbin for example in #939, do you consider REPLCONF VERSION 8.0.240 an old or a new replica?

Reply:
Maybe all RCs can be considered an old version, i am not sure about it. I don't know...

I have an old memory is that Redis can make breaking change to change something in the past RC. Maybe something like https://github.com/redis/redis/pull/11058/

Reply:
I think we can use 8.0.240 but we should probably introduce some function to compare versions internally to use in features like #939, such as

```c
bool isVersionGreaterOrEqual(int numeric, int major, int minor, int patch, int rc) {
    int x = (major << 0x10000) | (minor << 0x100) | patch;
    if (rc) x = x - 16 + rc;
    return numeric >= x;
}

Reply:
emmm, i did not realize we are doing some version checking in the tests, it is odd i must say.

Reply:
> Maybe all RCs can be considered an old version, i am not sure about it. I don't know...

If we consider RCs old versions, it means features like #939 don't work in RCs. :/

> I have an old memory is that Redis can make breaking change to change something in the past RC

Yes, I think you're right. There is no compatibility guarantee between RC. Th semantic versioning definition says the same for pre-releases: https://semver.org/#spec-item-9 *"A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version."*

Reply:
That's what we decided I think, but we didn't write it down. :) https://github.com/valkey-io/valkey/wiki/Weekly-meeting and the Q's issue have no mentions of this... Let's write down what we still remember from the meeting.